tesla_powerwall/api.py

```diff
@@ -128,12 +128,14 @@ def post(
         payload: dict,
         headers: dict = {},
     ) -> dict:
+        request_headers = {"Content-Type": "application/json"}
+        request_headers.update(headers)
```

Comment:
```suggestion
```

As far as I understood the request documentation the content type header will be automatically set, if the content is given via the json parameter.

Reply:
Done!  And re-ran the integration tests, looking good :)